common/import_.py

```diff
@@ -132,6 +132,15 @@ def link_technosphere_by_activity_hash_ref_product(
     )
 
 
+def search_activity(base_db, activity):
+    """Search for an activity in a database. If the activity contains "::", then the first element is the database name and the second is the activity name. Otherwise, the search is performed in the database specified in base_db."""
+    if "::" in activity:
+        db_name, activity_name = activity.split("::")
```

Comment:
What if you have two `::` in your string? If the activity name contains `::` for example, how will it behave?